worlds/yoshisisland/Items.py

```diff
@@ -0,0 +1,124 @@
+from typing import Dict, Set, Tuple, NamedTuple
+
+class ItemData(NamedTuple):
+    category: str
+    code: int
+    amount: int = 1

```

Comment:
```suggestion
    amount: Optional[int, None] = 1

```
Not entirely sure if this is how it is supposed to be written out. In several spots, you have a None in this field.